libvcx/src/aries/handlers/proof_presentation/verifier/state_machine.rs

```diff
@@ -87,22 +87,8 @@ impl VerifierSM {
             VerifierState::Initiated(state) => {
                 match message {
                     VerifierMessages::SendPresentationRequest => {
-                        let my_did = get_pw_did(connection_handle)?;
-                        let remote_did = get_their_pw_verkey(connection_handle)?;
-
-                        let presentation_request: PresentationRequestData =
-                            state.presentation_request_data.clone()
-                                .set_format_version_for_did(&my_did, &remote_did)?;
-
-                        let title = format!("{} wants you to share {}",
-                                            settings::get_config_value(settings::CONFIG_INSTITUTION_NAME)?, presentation_request.name);
-
-                        let presentation_request =
-                            PresentationRequest::create()
-                                .set_comment(title)
```

Comment:
We could keep this feature if parametrize `SendPresentationRequest` of `VerifierMessages` enum. Something like:
```
pub enum VerifierMessages {
    SendPresentationRequest(comment),
    ...
    }
    ```
 And so we could pass the comment variable from upper layers instead of having the logic hardcoded here (as it was until now)

Reply:
OK, it can be actually quite useful when not hardcoded